src/userSurvey.js

```diff
@@ -15,7 +15,7 @@
  * 
  * # Limitations
  * Note that this module is currently very limited: it only supports
- * one survey per study, with few options and a constrained design.
+ * one survey at a time per study, with few options and a constrained design.
```

Comment:
"the user complete the survey" → "the user to complete the survey" above. Would also point the reader to the  the documentation for the exported functions.